### PR TITLE
initdb: Add new page for initdb

### PR DESCRIPTION
**Pull request by MansoorMajeed**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/initdb.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# initdb
+
+> Create a PostgreSQL database on Disk.
+> More information: <https://www.postgresql.org/docs/9.5/app-initdb.html>.
+
+- Create a database at /usr/local/var/postgres:
+
+`initdb -D /usr/local/var/postgres`:
```
Comment (schneiderl) [inline pages/common/initdb.md]:
```suggestion
`initdb -D /usr/local/var/postgres`
```

File: pages/common/initdb.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# initdb
+
+> Create a PostgreSQL database on Disk.
```
Comment (zdroid) [inline pages/common/initdb.md]:
```suggestion
> Create a PostgreSQL database on disk.
```

File: pages/common/initdb.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# initdb
+
+> Create a PostgreSQL database on disk.
+> More information: <https://www.postgresql.org/docs/9.5/app-initdb.html>.
+
+- Create a database at /usr/local/var/postgres:
```
Comment (agnivade) [inline pages/common/initdb.md]: Perhaps we can change it to - "Create a database at a given location." ? I think that is simple enough.

File: pages/common/initdb.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+`initdb -D /usr/local/var/postgres`
```
Comment (agnivade) [inline pages/common/initdb.md]: Please wrap the location within backticks (``)

Comment (sbrl) [reply pages/common/initdb.md]: Shouldn't that be `{{token_syntax}}`, @agnivade?

Comment (agnivade) [reply pages/common/initdb.md]: My bad. Yes it should be the token syntax

Comment (sbrl) [reply pages/common/initdb.md]:
```suggestion
`initdb -D {{/usr/local/var/postgres}}`
```

(also, is there a long form of that flag?

**Pull request merged**